IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -901,12 +919,24 @@ protected
   parameter Modelica.SIunits.Area ACei(fixed=false);
 
   final parameter Integer indWalA = if hasNoA then 0 else 1;
-  final parameter Integer indWalB = indWalA + (if hasNoB then 0 else 1);
-  final parameter Integer indWalC = indWalB + (if hasNoC then 0 else 1);
-  final parameter Integer indWalD = indWalC + (if hasNoD then 0 else 1);
-  final parameter Integer indFlo = indWalD + (if hasNoFlo then 0 else 1);
-  final parameter Integer indCei = indFlo + (if hasNoCei then 0 else 1);
-  final parameter Integer indIntZone_a = indCei + (if hasInt then 1 else 0);
+  final parameter Integer indWalA_end = indWalA + (if not hasNoA and bouTypA == IDEAS.Buildings.Components.Interfaces.BoundaryType.External
```

Comment:
`bouTypA == IDEAS.Buildings.Components.Interfaces.BoundaryType.External` already implies that `hasNoA=true` so why did you add `not hasNoA and`?

Reply:
correct. Done